modules/hcp-dc-image-mirror.adoc

```diff
@@ -0,0 +1,98 @@
+
+To configure image mirroring, complete the following steps:
+
+. Ensure that your `${HOME}/.docker/config.json` file is updated with the registries that you are going to mirror from and with the private registry that you plan to push the images to.
+
+. By using the following example, create an `ImageSetConfiguration` object to use for mirroring. Replace values as needed to match your environment:
++
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
+  registry:
+    imageURL: registry.<dns.base.domain.name>:5000/openshift/release/metadata:latest <1>
+mirror:
+  platform:
+    channels:
+    - name: candidate-4.17
+      minVersion: 4.x.y-x86_64  <2>
+      maxVersion: 4.x.y-x86_64 <2>
+      type: ocp
+    kubeVirtContainer: true <3>
+    graph: true
+  additionalImages:
+  - name: quay.io/karmab/origin-keepalived-ipfailover:latest
+  - name: quay.io/karmab/kubectl:latest
+  - name: quay.io/karmab/haproxy:latest
+  - name: quay.io/karmab/mdns-publisher:latest
+  - name: quay.io/karmab/origin-coredns:latest
+  - name: quay.io/karmab/curl:latest
+  - name: quay.io/karmab/kcli:latest
+  - name: quay.io/user-name/trbsht:latest
+  - name: quay.io/user-name/hypershift:BMSelfManage-v4.17
+  - name: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.10
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.17
+    packages:
+    - name: lvms-operator
+    - name: local-storage-operator
+    - name: odf-csi-addons-operator
+    - name: odf-operator
+    - name: mcg-operator
+    - name: ocs-operator
+    - name: metallb-operator
+    - name: kubevirt-hyperconverged <4>
+----
++
+<1> Replace `<dns.base.domain.name>` with the DNS base domain name.
+<2> Replace `4.x.y` with the supported {product-title} version you want to use.
+<3> Set this optional flag to `true` if you want to also mirror the container disk image for the {op-system-first} boot image for the KubeVirt provider. This flag is available with oc-mirror v2 only.
+<4> For deployments that use the KubeVirt provider, include this line.
+
+. Start the mirroring process by entering the following command:
++
+[source,terminal]
+----
+$ oc-mirror --v2 --source-skip-tls --config imagesetconfig.yaml docker://${REGISTRY}
+----
++
+After the mirroring process is finished, you have a new folder named `oc-mirror-workspace/results-XXXXXX/`, which contains the IDMS and the catalog sources to apply on the hosted cluster.
+
+. Mirror the nightly or CI versions of {product-title} by using the `oc adm release mirror` command. Enter the following command:
++
+[source,terminal]
+----
+REGISTRY=registry.$(hostname --long):5000
+
+$ oc adm release mirror \
+  --from=registry.ci.openshift.org/ocp/release:4.x.y-x86_64 \ <1>
+  --to=${REGISTRY}/openshift/release \
+  --to-release-image=${REGISTRY}/openshift/release-images:4.x.y-x86_64 <1>
+----
++
+<1> Replace `4.x.y` with the supported {product-title} version you want to use.
```

Comment:
this is just an alternative to the previous method based on `oc-mirror`:
having something like:
```
apiVersion: mirror.openshift.io/v2alpha1
kind: ImageSetConfiguration
mirror:
  platform:
    channels:
    - name: candidate-4.17
      minVersion: 4.x.y-x86_64  <2>
      maxVersion: 4.x.y-x86_64 <2>
```
will cause oc-mirror to inspect the cincinnati APIs to select the OCP release image according to that.

But you can also use something like:
```
apiVersion: mirror.openshift.io/v2alpha1
kind: ImageSetConfiguration
mirror:
  platform:
    graph: true
    release: registry.ci.openshift.org/ocp/release:4.x.y-x86_64
    kubeVirtContainer: true
```

to directly point to a specific nightly/CI image.
I think we should mention there instead of introducing an alternative mechanism that could confuse the user.